lib/open_food_network/bulk_coop_report.rb

```diff
@@ -134,5 +140,127 @@ def order_permissions
     def report_line_items
       @report_line_items ||= Reports::LineItems.new(order_permissions, @params)
     end
+
+    def customer_payments_total_cost(line_items)
```

Comment:
All these methods below could be extracted to a service. Some of them are repeated in other reports I am sure.